src/cephadm/cephadm

```diff
@@ -3254,6 +3254,10 @@ class CephContainer:
         self._cname = cname
         self.container_args = container_args
         self.envs = envs
+        if "ceph-volume" in self.entrypoint:
+            if self.envs is None:
+                self.envs = []
+            self.envs.append("CEPH_VOLUME_SKIP_RESTORECON=yes")
```

Comment:
this feels like a wrong location for it. 

We have:

https://github.com/ceph/ceph/blob/0f7791fa24a07d543fb5a2de11c15c3b4c01dccd/src/cephadm/cephadm#L2492

already. I think we need a `def get_ceph_volume_container()` to encapsulate things like 

https://github.com/ceph/ceph/blob/0f7791fa24a07d543fb5a2de11c15c3b4c01dccd/src/cephadm/cephadm#L5783-L5789

as well. Or

https://github.com/ceph/ceph/blob/0f7791fa24a07d543fb5a2de11c15c3b4c01dccd/src/cephadm/cephadm#L6141-L6149